Build Failure: javax.xml.bind.annotation does not exist

# Description
keytrans-server 0.2-SNAPSHOT fails to build due to missing files. 

## Error Message:
```sh
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:compile (default-compile) on project keytrans-server: Compilation failure: Compilation failure: 
[ERROR] /home/jason/src/TASQC/gov.ornl.tasqc.keytrans/src/gov/ornl/tasqc/keytrans/QuantumKey.java:[35,32] error: package javax.xml.bind.annotation does not exist
```

## Possible Explanation:
These files are deprecated and have been removed from JAVA 11:
[JEP 320: Remove the Java EE and CORBA Modules](http://openjdk.java.net/jeps/320#Java-EE-modules)

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,2 @@
+gov.ornl.tasqc.keytrans/static/
+keyTransferClient-build/

--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ From the base directory where you checked out the repository:
 ```sh
 mkdir keyTransferClient-build
 cd keyTransferClient-build
-cmake ../keyTransferClient-build
+cmake ../keyTransferClient
 ```
 
 You may pick any build directory you want, actually. Just replace `../` with the directory.

--- a/gov.ornl.tasqc.keytrans/dependency-reduced-pom.xml
+++ b/gov.ornl.tasqc.keytrans/dependency-reduced-pom.xml
@@ -25,6 +25,33 @@
       <plugin>
         <artifactId>maven-shade-plugin</artifactId>
         <version>2.3</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <transformers>
+                <transformer />
+                <transformer />
+              </transformers>
+            </configuration>
+          </execution>
+        </executions>
+        <configuration>
+          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <filters>
+            <filter>
+              <artifact>*:*</artifact>
+              <excludes>
+                <exclude>META-INF/*.SF</exclude>
+                <exclude>META-INF/*.DSA</exclude>
+                <exclude>META-INF/*.RSA</exclude>
+              </excludes>
+            </filter>
+          </filters>
+        </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
@@ -41,6 +68,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-site-plugin</artifactId>
+        <version>3.7.1</version>
         <configuration>
           <inputEncoding>UTF-8</inputEncoding>
           <outputEncoding>UTF-8</outputEncoding>

--- a/gov.ornl.tasqc.keytrans/pom.xml
+++ b/gov.ornl.tasqc.keytrans/pom.xml
@@ -35,6 +35,21 @@
 			<version>4.11</version>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-core</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.3.0</version>
+        </dependency>
 	</dependencies>
 
 	<build>
@@ -107,6 +122,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-site-plugin</artifactId>
+                <version>3.7.1</version>
 				<configuration>
 					<inputEncoding>UTF-8</inputEncoding>
 					<outputEncoding>UTF-8</outputEncoding>

--- a/keyTransferClient/libcurlUtil/CMakeLists.txt
+++ b/keyTransferClient/libcurlUtil/CMakeLists.txt
@@ -40,7 +40,7 @@ SET(LIBRARY_NAME "curlUtil")
 
 #Collect all header filenames in this project 
 #and glob them in HEADERS
-file(GLOB HEADERS *.h)
+file(GLOB HEADERS *.h *.hpp)
 
 #Grab all of the source files
 file(GLOB SRC *.cpp)

